kubenow/gluster-s3/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for a Gluster-S3 deployment.
+# this is a yaml-formatted file.
+# declare variables to be passed into your templates.
+releaseName: s3-interface
+secretName: s3-secret
```

Comment:
no need to have a name for the secret, it can be hardcoded

Reply:
these is a good practice in Helm afaik